### PR TITLE
blocks and tasks update

### PR DESCRIPTION
**Pull request by csidyel**

### CODE REVIEW

File: docs/guided-tour/concepts.md [hunk 1]
```diff
@@ -1,27 +1,36 @@
...
+sourced from `.semaphore/semaphore.yml`. Additional pipelines triggered via
+promotions are defined in separate files.
 
 ![Semaphore 2.0 concepts diagram](https://storage.googleapis.com/semaphore-public-assets/public/images/semaphoreci2-concepts.png)
 
 ## Blocks & Tasks
 
-_Blocks_ are the heart of a pipeline. Each block has a _task_ that
-defines one or more _jobs_. Jobs define the commands to execute.
+Blocks are the building blocks of a pipeline. Each block has a _task_ defined by
```
Comment (csidyel) [inline docs/guided-tour/concepts.md:20]: @markoa This first sentence looks weird to me wdyt?

Comment (markoa) [reply docs/guided-tour/concepts.md:20]: Why specifically?

Comment (csidyel) [reply docs/guided-tour/concepts.md:20]: To me, it sounds weird that such a short sentence defines Blocks by being blocks.

Comment (markoa) [reply docs/guided-tour/concepts.md:20]: I see, I don't think I've realized the duplication until now. I'll think of something else async.

Comment (csidyel) [reply docs/guided-tour/concepts.md:20]: I've thought about this one and probably the best idea that comes to mind is:
`Blocks are the building steps of a pipeline.`
The word step is already used above:
`_Blocks_ define what to do at each step in the pipeline.`
Anything else I can think of changes the sentence almost entirely.

**Comment by Ivana-Markovic** [comment]
I see that this is merged but just wanted to share that in the above examples, blocks can be referred to as units (maybe 🙂): "Blocks are the building **units** of a pipeline."

**Pull request merged**